performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -0,0 +1,54 @@
+Firebase Performance Monitoring Quickstart
+=============================
+
+# Firebase Performance Monitoring
+
+Firebase Performance Monitoring is a free mobile app performance analytics service. It
+provides detailed information about the performance of your apps automatically,
+but works at its best with Timers set by you.
+
+Introduction
+------------
+
+- [Read more about Firebase Performance Monitoring](https://firebase.google.com/docs/perf-mon/)
+
+Getting Started
+---------------
+
+- [Add Firebase to your iOS Project](https://firebase.google.com/docs/ios/setup).
+- [Create a default Cloud Storage bucket](https://firebase.google.com/docs/storage/ios/start#create-default-bucket).
+- [Replace your Rules](https://firebase.google.com/docs/storage/security/get-started#access_your_rules) with the following:
+```
+rules_version = '2';
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /image.jpg {
+      allow create: if request.resource.contentType.matches('image/jpeg')
+    }
+  }
+}
+```
```

Comment:
The UI tests are failing because it is unable to test the upload functionality, which I assume is happening because of the rules of the CI Firebase project. Should the upload functionality not be tested? Would I be able to use different rules to test the upload functionality with CI?

Reply:
To clarify, the tests pass locally on iOS 14 & 15 and tvOS 14 & 15.

Reply:
Gave you editor access to the project so you can change the rules.